frontend/packages/ux-editor/src/components/Preview/Preview.tsx

```diff
@@ -27,6 +28,11 @@ export const Preview = () => {
         title={t('ux_editor.preview')}
         src={previewPage(org, app, selectedLayoutSet)}
       />
+      <div className={classes.previewLimitationsInfo}>
+        <Alert severity='info'>
+          {t('preview.limitations_info')}
+        </Alert>
+      </div>
```

Comment:
Ref. forrige kommentar - så kan vi nå fjerne container-elementet og sende CSS-klassen direkte ned til Alert istedenfor. 😊 
```suggestion
        <Alert severity='info' className={classes.previewLimitationsInfo}>
          {t('preview.limitations_info')}
        </Alert>
```